exif2timestream.py

```diff
@@ -581,6 +689,11 @@ def main(opts):
     # beginneth the actual main loop
     start_time = time()
     cameras = parse_camera_config_csv(opts["-c"])
+    try:
+        global EXIF_DATE_MASK
+        EXIF_DATE_MASK = opts["-m"]
```

Comment:
Check this behaviour by doing `print(opts)` with and without the `-m` param. I think it will still be in the dict, and the if statement will work. But i'm possibly wrong.